app-crypt/veracrypt/files/veracrypt-1.21-remove-packaging-from-makefile.patch

```diff
@@ -0,0 +1,107 @@
+--- src/Main/Main.make.old	2017-08-02 01:55:38.841516167 +0300
++++ src/Main/Main.make	2017-08-02 01:58:29.479077681 +0300
+@@ -219,104 +219,6 @@
+ 	rm -fr "$(PWD)/Setup/MacOSX/VeraCrypt_dmg"
+ endif
+ 
+-
+-
+-ifeq "$(PLATFORM)" "Linux"
```

Comment:
Why are you removing this? Unless I'm missing something, it won't be run anyway since you don't call the `install` nor `prepare` target.